packages/teams-js/src/public/app.ts

```diff
@@ -63,88 +67,93 @@ export function appInitializeHelper(apiVersionTag: string, validMessageOrigins?:
+
+        // Teams AppHost won't provide this runtime config
+        // so we assume that if we don't have it, we must be running in Teams.
+        // After Teams updates its client code, we can remove this default code.
+        try {
+          initializeHelperLogger('Parsing %s', runtimeConfig);
+          const givenRuntimeConfig: IBaseRuntime | null = JSON.parse(runtimeConfig);
+          initializeHelperLogger('Checking if %o is a valid runtime object', givenRuntimeConfig ?? 'null');
+          // Check that givenRuntimeConfig is a valid instance of IBaseRuntime
+          if (!givenRuntimeConfig || !givenRuntimeConfig.apiVersion) {
+            throw new Error('Received runtime config is invalid');
+          }
+          runtimeConfig && applyRuntimeConfig(givenRuntimeConfig);
+        } catch (e) {
+          if (e instanceof SyntaxError) {
+            try {
+              initializeHelperLogger('Attempting to parse %s as an SDK version', runtimeConfig);
+              // if the given runtime config was actually meant to be a SDK version, store it as such.
+              // TODO: This is a temporary workaround to allow Teams to store clientSupportedSDKVersion even when
+              // it doesn't provide the runtimeConfig. After Teams updates its client code, we should
+              // remove this feature.
+              if (!isNaN(compareSDKVersions(runtimeConfig, defaultSDKVersionForCompatCheck))) {
+                GlobalVars.clientSupportedSDKVersion = runtimeConfig;
+              }
+              const givenRuntimeConfig: IBaseRuntime | null = JSON.parse(clientSupportedSDKVersion);
+              initializeHelperLogger('givenRuntimeConfig parsed to %o', givenRuntimeConfig ?? 'null');
+
+              if (!givenRuntimeConfig) {
+                throw new Error(
+                  'givenRuntimeConfig string was successfully parsed. However, it parsed to value of null',
+                );
+              } else {
+                applyRuntimeConfig(givenRuntimeConfig);
+              }
+            } catch (e) {
+              if (e instanceof SyntaxError) {
+                applyRuntimeConfig(
+                  generateVersionBasedTeamsRuntimeConfig(
+                    GlobalVars.clientSupportedSDKVersion,
+                    versionAndPlatformAgnosticTeamsRuntimeConfig,
+                    mapTeamsVersionToSupportedCapabilities,
+                  ),
+                );
+              } else {
+                throw e;
               }
-            } else {
-              // If it's any error that's not a JSON parsing error, we want the program to fail.
-              throw e;
             }
+          } else {
+            // If it's any error that's not a JSON parsing error, we want the program to fail.
+            throw e;
           }
+        }
 
-          GlobalVars.initializeCompleted = true;
-        },
-      );
+        GlobalVars.initializeCompleted = true;
+      });
```

Comment:
This does look like it's changed. Curious if you can grab what this should be from `main` branch and copy it back into the file.

Reply:
I tried that twice. There must be some setting in vscode that I need to figure

Reply:
It looks fine locally - just the indentation had changed, but here (on github), it looks like I have changed the content

![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/88564543/9da47fa9-877f-4008-9aa7-7a5f9bee0b27)